src/index.html

```diff
@@ -10,13 +10,54 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"class="logo">
+        <img
+          src="/src/images/logo.png" alt="navigationlogo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a data-qa="hover" class="nav__link">laptops & computers</a>
```

Comment:
Avoid using spaces in the `href` property of an `<a>` tag. The value 'laptops & computers' should not contain spaces. Use a dash or underscore instead.